dms-core/src/main/kotlin/team/aliens/dms/domain/outing/dto/OutingResponse.kt

```diff
@@ -50,3 +53,48 @@ data class OutingApplicationHistoriesResponse(
 data class OutingAvailableTimesResponse(
     val outingAvailableTimes: List<OutingAvailableTimeVO>
 )
+
+data class OutingHistoryDetailsResponse(
+    val outingTime: LocalTime,
+    val arrivalTime: LocalTime,
+    val outingStatus: OutingStatus,
+    val reason: String?,

```

Comment:
```suggestion
    @field:JsonInclude(JsonInclude.Include.ALWAYS)
    val reason: String?,
```
null일 때도 반환되도록 변경해주세요.